### ISSUE TITLE
🐛 Bug: Incorrect API call in AutomatedTests.ipynb

### ISSUE THREAD

**Issue by pavithranair** [ersilia-os/ersilia#524]
### Describe the bug.

The API call in [AutomatedTests.ipynb](https://github.com/ersilia-os/ersilia/blob/master/AutomatedTests.ipynb) is passing a python variable to a terminal command. Hence, the variable name is being treated as a string and the following error is encountered:

![image](https://user-images.githubusercontent.com/66196840/211069205-c823bdb1-2361-47ff-99b0-5c60f76359d9.png)

The command causing the error:
`!ersilia api -i smi`

### Describe the steps to reproduce the behavior

1. Open [AutomatedTests.ipynb](https://github.com/ersilia-os/ersilia/blob/master/AutomatedTests.ipynb) in Colab
2. Run the cells

### Screenshots.

![image](https://user-images.githubusercontent.com/66196840/211069151-313dfa17-7bc9-43c1-be1c-edb3e9fa49e2.png)

### Operating environment

Ubuntu 18.04.2 lts

**Comment by GemmaTuron** [comment]
Hi @pavithranair !

Thanks, this notebook for automated testing of models in Google Colab is still work in progress so many of the inputs still need work, we are working with @miquelduranfrigola and @megamanics.
I'll merge this commit, and I'd suggest waiting until we have the final version ready before testing it again ;)

**Comment by GemmaTuron** [comment]
Hi,

Just to comment on this for clarification. Sorry I rushed through the comments and did not check properly. We have now edited the different notebooks we will be using:
- ersilia-on-colab (for general users, a simple form-based colab notebook without code exposed)
- test-ersilia ...: automated tests that will be run everytime we make changes to ersilia's code

The cange that @pavithranair was suggesting is actually not ideal, since we want to pass smiles as strings. The error you were encountering was due to a wrong smiles format. I've kept the old notation for the test notebooks and the main colab notebook
